python/private/toolchains_repo.bzl

```diff
@@ -195,6 +196,9 @@ load("{rules_python}//python:pip.bzl", _compile_pip_requirements = "compile_pip_
 
 host_platform = "{host_platform}"
 interpreter = "@{py_repository}_{host_platform}//:{python3_binary_path}"
+PLATFORMS = [
```

Comment:
Please take this out of defs.bzl -- defs.bzl is a public entry point, and this isn't a detail we want to make part of our public API.

Instead, it can be inlined in build_contents directly:
```
_plaforms = [
  {loaded_platforms}
]
```

Reply:
IMO the selection of platforms is valuable information, but I am not sure if this is the right form to make that available. So, I am fine with removing it.

Reply:
Changed as suggested.